tests/test_trackingstore.py

```diff
@@ -330,6 +330,60 @@ def test_create_run(mocker, mlflow_parent_run_id, faculty_parent_run_id):
     assert returned_run == mlflow_run
 
 
+def test_create_run_experiment_deleted(mocker):
+    mlflow_timestamp = mocker.Mock()
+    faculty_datetime = mocker.Mock()
+    timestamp_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore."
+        "mlflow_timestamp_to_datetime_milliseconds",
+        return_value=faculty_datetime,
+    )
+
+    mlflow_tag = mocker.Mock()
+    faculty_tag = mocker.Mock()
+    tag_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore.mlflow_tag_to_faculty_tag",
+        return_value=faculty_tag,
+    )
+
+    mock_client = mocker.Mock()
+    exception = faculty.clients.experiment.ExperimentDeleted(
+        message="message", experiment_id="test-id"
+    )
+
+    mock_client.create_run.side_effect = exception
+    mocker.patch("faculty.client", return_value=mock_client)
+    experiment_id = mocker.Mock()
+    run_name = mocker.Mock()
+
+    store = FacultyRestStore(STORE_URI)
+
+    with pytest.raises(MlflowException, match="experiment"):
+        store.create_run(
+            experiment_id,
+            "unused-mlflow-user-id",
+            run_name,
+            "unused-source-type",
+            "unused-source-name",
+            "unused-entry-point-name",
+            mlflow_timestamp,
+            "unused-source-version",
+            [mlflow_tag],
+            PARENT_RUN_UUID_HEX_STR,
+        )
+
+    timestamp_converter_mock.assert_called_once_with(mlflow_timestamp)
```

Comment:
We often omit these 'called with' assertions in failure cases - these are checked by the happy path test.

I recommend removing these, which should also allow you to not bind e.g. `tag_converter_mock` and others to variables above.